packages/deno/src/generators/setup-functions/setup-functions.ts

```diff
@@ -15,9 +14,10 @@ export async function denoSetupServerless(
     case 'netlify':
       task = addNetlifyConfig(tree, opts);
       break;
-    case 'deno-deploy':
-      task = await addDenoDeployConfig(tree, opts);
-      break;
+    // TODO: This should be a separate generator e.g. nx g @nx/deno:deploy
+    // case 'deno-deploy':
+    //   task = await addDenoDeployConfig(tree, opts);
+    //   break;
```

Comment:
There is a follow-up needed for `@nx/deno:deploy` generator, which we could call from the `@nx/deno:app` generator via a `--deno-deploy` flag or something? It makes sense to have Deno Deploy option there I think since it is official.